docs/building-a-form/common-definitions.md

```diff
@@ -171,7 +142,7 @@ A function that takes a full name `schema` and sets its required field list to `
 - `schema`: Yes
 - Function or object: Function
 
-##### Person ID
+### Person ID
 
 A function that allows the user to input a form of personal identification. For example, the form can ask for a social security number and include a checkbox to specify that the user doesn't have one, allowing them to instead input a Veterans Affairs file number.
```

Comment:
Is *input* verbable nowadays?

Reply:
As long as people understand what this means I'm fine with it. 🤷‍♀️

Reply:
Ugh, I'll change it to "type".